src/project-1.js

```diff
@@ -1,6 +1,7 @@
 // Do not change any of the funcxtion names
 
 const multiplyByTen = (num) => {
```

Comment:
Here, you could write something like
`const multiplyByTen = num => num * 10` with an implicit return.